aptos-move/block-executor/src/code_cache_global_manager.rs

```diff
@@ -290,13 +290,20 @@ fn prefetch_aptos_framework(
     let code_storage = state_view.as_aptos_code_storage(guard.environment());
 
     // If framework code exists in storage, the transitive closure will be verified and cached.
-    let maybe_loaded = code_storage
-        .fetch_verified_module(&AccountAddress::ONE, ident_str!("transaction_validation"))
-        .map_err(|err| {
-            // There should be no errors when pre-fetching the framework, if there are, we
-            // better return an error here.
-            PanicError::CodeInvariantError(format!("Unable to fetch Aptos framework: {:?}", err))
-        })?;
+    let maybe_loaded = {
+        #[cfg(fuzzing)]
+        let result =
+            code_storage.fetch_module(&AccountAddress::ONE, ident_str!("transaction_validation"));
+        #[cfg(not(fuzzing))]
+        let result = code_storage
+            .fetch_verified_module(&AccountAddress::ONE, ident_str!("transaction_validation"));
```

Comment:
This no longer prefetches framework? Is this intended? `fetch_module` will stop at top level module so only `transaction_validation` will be cached. You might just skip framework caching here altogether as a short term solution? 

(the caller of `prefetch_aptos_framework` checks if cache is empty and config to prefetch is on, so you can either modify the config for fake executor or just gate with `#[cfg(not(fuzzing))]` to avoid pre-fetch?